lightning/src/chain/onchaintx.rs

```diff
@@ -623,19 +624,31 @@ impl<ChannelSigner: WriteableEcdsaChannelSigner> OnchainTxHandler<ChannelSigner>
 			return inputs.find_map(|input| match input {
 				// Commitment inputs with anchors support are the only untractable inputs supported
 				// thus far that require external funding.
-				PackageSolvingData::HolderFundingOutput(..) => {
+				PackageSolvingData::HolderFundingOutput(output) => {
 					debug_assert_eq!(tx.txid(), self.holder_commitment.trust().txid(),
 						"Holder commitment transaction mismatch");
+
+					let conf_target = ConfirmationTarget::HighPriority;
+					let package_target_feerate_sat_per_1000_weight = cached_request
+						.compute_package_feerate(fee_estimator, conf_target, force_feerate_bump);
+					if let Some(input_amount_sat) = output.funding_amount {
+						let fee_sat = input_amount_sat - tx.output.iter().map(|output| output.value).sum::<u64>();
+						if compute_feerate_sat_per_1000_weight(fee_sat, tx.weight() as u64) >=
+							 package_target_feerate_sat_per_1000_weight
+						{
+							log_debug!(logger, "Commitment transaction {} already meets required feerate {} sat/kW",
```

Comment:
Log can be clearer like “Pre-signed feerate of the commitment transaction {} already meets feerate required by feestimator {} sat/kW”.